src/bindings/event.ts

```diff
@@ -17,14 +17,8 @@ export const getBotConfig = () => botConfig;
 let adapters: Adapters = {} as Adapters;
 export const getAdapters = () => adapters;
 
-export type Logger = {
-  info: (msg: string | object, options?: JSON) => void;
-  debug: (msg: string | object, options?: JSON) => void;
-  warn: (msg: string | object, options?: JSON) => void;
-  error: (msg: string | object, options?: JSON) => void;
-};
```

Comment:
This is an interface that we can integrate any kind of logging libraries. 
Even if you create a GithubLogger, it should be inherited from `Logger`.

Reply:
Some libraries like `wiston` does not do the log.info  or debug thing though

But it'd bring that back and inherit from it